### ISSUE TITLE
Transformer forward pass

### ISSUE THREAD

**Issue by max-andreasen** [HaraldHR/NLP_Project#8]
Is this needed for the requirements? Can't use torch.nn.transformer? 
* Implement the forward pass for the transformer neural network model and return the loss object from torch.
